.github/workflows/deploy-qa.yml

```diff
@@ -0,0 +1,93 @@
+name: Publish images
```

Comment:
Rename this to differentiate from the "Publish images" action.  @advcoder 
Also it's failing: https://github.com/department-of-veterans-affairs/abd-vro/actions/workflows/deploy-qa.yml